src/utils/data-source.js

```diff
@@ -37,9 +37,9 @@ export const getSourceID = () => {
  */
 export const getDataValue = source => {
   switch (source) {
-    case 'animals':
-      // Use data from the 'animals' test dataset
-      return animals;
+    case 'testData':
+      // Use data from the 'testData' test dataset
+      return testData;
```

Comment:
hmm. I'm not stoked that this has a capital letter in it - I think ideally we'd want to keep URLs in lowercase. Anyway I'd be inclined to use 'test' as the datasource name anyway.

It's something I've been pondering through this renaming process - i.e. whether I'd prefer for it to just be called 'test' rather than 'testData', because that should be reasonably clear already as it's in the 'data' directory. e.g. 'demo' doesn't need to be called 'demoData', because it's clear that 'data/demo.json' is a data file. Furthermore, 'testData' gets confusing when you have multiple test data files, or if we want to add another in the future. And 'testData' doesn't describe the contents. Finally, changing the URL in the source effectively means we're changing the external-facing developer experience - it's a very minor breaking change. Not something to be done lightly.

Let's discuss this tomorrow and see what we think.

Reply:
I agree with your concerns - definitely on board with your comment on keeping URLs in lowercase. While I was modifying this file it did come to my mind that calling this as the `test-data` dataset is a bit redundant as everything we refer to here are 'data' - I would say within the tests, it does make sense to call this as `testData` as calling it as `test` within tests is confusing. 

Perhaps the best way forward is to call this as `test` instead of `test-data` within `data-source.js`, `app.js` and the contributing docs, while keeping it as `testData` within the tests for clarify purposes?